evm-contracts/src/v0.6-ovm/v0.6-ovm.diff

```diff
@@ -0,0 +1,1500 @@
+diff --git a/src/v0.6/AccessControlledAggregator.sol b/src/v0.6-ovm/AccessControlledAggregator.sol
```

Comment:
Is this file needed?

Reply:
Probably not in the long term. For now, I've included it as it might help assess the changes between our `v6` contracts and the new `v6-ovm` for audit purposes.